Commit [r8061]  Maximize  History

[GB.QT5.EXT]
* BUG: The component now should compile correctly with C++11.
Authored by:  gambas 3 days ago
Browse code at this revision
Parent: [r8060]
changed	/gambas/trunk/gb.qt5/configure.ac
changed	/gambas/trunk/gb.qt5/src/ext/Makefile.am
/gambas/trunk/gb.qt5/configure.ac Diff Switch to side-by-side view
--- a/gambas/trunk/gb.qt5/configure.ac
+++ b/gambas/trunk/gb.qt5/configure.ac
@@ -14,11 +14,6 @@
   qt5, QT5, gb.qt5, [src],
   'Qt5Core >= 5.3.0' Qt5Gui Qt5Widgets Qt5Svg Qt5PrintSupport Qt5X11Extras x11
 )
-
-## GB_COMPONENT_PKG_CONFIG(
-##   qt5ext, QT5EXT, gb.qt5.ext, [ext],
-##   'Qt5Core >= 5.4.0' Qt5Gui Qt5Widgets
-## )
 
 GB_COMPONENT_PKG_CONFIG(
   qt5webkit, QT5WEBKIT, gb.qt5.webkit, [webkit],
@@ -55,6 +50,4 @@
 src/ext/Makefile \
 )
 
-## src/ext/Makefile \
-
 GB_PRINT_MESSAGES
--- a/gambas/trunk/gb.qt5/src/ext/Makefile.am
+++ b/gambas/trunk/gb.qt5/src/ext/Makefile.am
@@ -6,6 +6,7 @@
 
 gb_qt5_ext_la_LIBADD = @QT5EXT_LIB@
 gb_qt5_ext_la_LDFLAGS = -module @LD_FLAGS@ @QT5EXT_LDFLAGS@
+gb_qt5_ext_la_CXXFLAGS = @THREAD_INC@ $(AM_CXXFLAGS) -std=c++11
 gb_qt5_ext_la_CPPFLAGS = @QT5EXT_INC@ -I$(top_srcdir)/share -I$(top_srcdir)/src/share
 
 gb_qt5_ext_la_SOURCES = \
